deploy/role.yaml

```diff
@@ -4,110 +4,84 @@ metadata:
   creationTimestamp: null
   name: kogito-operator
 rules:
```

Comment:
The operator-sdk rewrote the RBAC, and this will jeopardize our installation. Could you please rollback these changes? Just make sure that the `KogitoTrusty` is included among the types.

Reply:
I've reverted it, could you double check that everything is right?